recipe/meta.yaml

```diff
@@ -35,10 +35,11 @@ requirements:
     - pytorch
     - pytorch =*={{ torch_proc_type }}*
     - pytest-runner
-    - libcusparse-dev    # [(cuda_compiler_version or "").startswith("12")]
-    - libcublas-dev      # [(cuda_compiler_version or "").startswith("12")]
-    - libcusolver-dev    # [(cuda_compiler_version or "").startswith("12")]
-    - libxcrypt          # [linux and py<39]
+    - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+    - libcusparse-dev                           # [(cuda_compiler_version or "").startswith("12")]
+    - libcublas-dev                             # [(cuda_compiler_version or "").startswith("12")]
+    - libcusolver-dev                           # [(cuda_compiler_version or "").startswith("12")]
+    - libxcrypt                                 # [linux and py<39]
```

Comment:
we don't build for python 3.8 anymore, so if you encounter selectors with `py<39`, you can just remove that line. :)
```suggestion
```